chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/endpoints/ChaosMonkeyJmxEndpoint.java

```diff
@@ -78,13 +79,15 @@ public String isChaosMonkeyActive() {
   @WriteOperation
   public String enableChaosMonkey() {
     this.chaosMonkeySettings.getChaosMonkeyProperties().setEnabled(true);
-    return "Chaos Monkey is enabled";
+    return "Chaos Monkey is enabled\nActivatedAt:"
+        + Clock.systemDefaultZone().instant().atZone(Clock.systemDefaultZone().getZone());
```

Comment:
Now that I see it, I would preferr the term "enabledAt" over "activatedAt" as we already use the verb enabled in "Chaos Monkey is enabled".


I think it would make sense to use StringJoiner, it would make the code more readable:
```java
new StringJoiner("\n")
            .add("Chaos Monkey is enabled")
            .add("enabledAt: " + ZonedDateTime.now())
            .toString();
 ````
 
However I think even better would be to introduce a new class (POJO, eg. ChaosMonkeyEnabledDto and ChaosMonkeyDisabledDto) and return that instead. Then we dont need to deal with "\n" and formats  and spring/jackson can take care of the conversion of that object to a string :)
 
 Then it could look like this:
 ```java
   @WriteOperation
  public ChaosMonkeyEnabledDto enableChaosMonkey() {
    this.chaosMonkeySettings.getChaosMonkeyProperties().setEnabled(true);
    return new ChaosMonkeyEnabledDto();
  }
 ```
 
 ```java
 @Data
public class ChaosMonkeyEnabledDto {
    private final String status = "Chaos Monkey is enabled";
    private ZonedDateTime enabledAt = ZonedDateTime.now();
}
```

Reply:
The class's is a good idea. I recall seeing ZonedDateTime.now(). Will need to check again to figure out why didn't go with it!